schemas/cloudflare/terraform-resource-1.yml

```diff
@@ -63,6 +63,17 @@ properties:
         type: boolean
       smart_routing:
         type: boolean
+  tiered_cache:
+    type: object
+    additionalProperties: false
+    properties:
+      cache_type:
+        type: string
+        enum:
+          - generic
+          - smart
```

Comment:
nit: is `off` not necessary: https://registry.terraform.io/providers/cloudflare/cloudflare/latest/docs/resources/tiered_cache
?

I don't know a lot of use cases immediately, but maybe something when debugging?

Reply:
Their API docs and terraform docs don't match up. I went with what they have available in the web console which is the two options (generic and smart). From what I can understand from looking at the API and provider code, `off` is taken care of if we set `argo.tiered_caching` to `off`